packages/grafana-ui/src/components/ValueMappingsEditor/ValueMappingsEditor.story.tsx

```diff
@@ -30,6 +30,19 @@ export function Example() {
         },
       },
     },
+    {
+      type: MappingType.RegexToText,
+      options: {
+        // TODO correct format?
```

Comment:
is it the correct format? :wink:

Reply:
As it turned out, yes :smile: 

Have removed the comments.